From 599043d78f38fd8faa822c607969f4f18b2b2ea6 Mon Sep 17 00:00:00 2001
From: Victor Popescu <victor.popescu@nxp.com>
Date: Wed, 29 Mar 2017 17:37:15 +0300
Subject: [PATCH 2/8] mac80211: Temporary timing workartound needed  to be run
 on simulator

this commit needs to be reverted when we will switch on hw

Signed-off-by: Alexandru Popescu alexandru.popescu@nxp.com
---
 net/mac80211/scan.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/net/mac80211/scan.c b/net/mac80211/scan.c
index 7bb6a93..d2955a1 100644
--- a/net/mac80211/scan.c
+++ b/net/mac80211/scan.c
@@ -210,7 +210,6 @@ void ieee80211_scan_rx(struct ieee80211_local *local, struct sk_buff *skb)
 
 	if (baselen > skb->len)
 		return;
-
 	ieee802_11_parse_elems(elements, skb->len - baselen, false, &elems);
 
 	channel = ieee80211_get_channel(local->hw.wiphy, rx_status->freq);
@@ -493,7 +492,7 @@ static void ieee80211_scan_state_send_probe(struct ieee80211_local *local,
 	 * After sending probe requests, wait for probe responses
 	 * on the channel.
 	 */
-	*next_delay = IEEE80211_CHANNEL_TIME;
+	*next_delay = IEEE80211_CHANNEL_TIME*20;
 	local->next_scan_state = SCAN_DECISION;
 }
 
-- 
2.1.0.27.g96db324

